plugwise_usb/connection/manager.py

```
@@ -0,0 +1,196 @@
+        self._receiver: StickReceiver | None = None
+        self._serial_transport: SerialTransport | None = None
+        self._port = "<not defined>"
+        self._connected: bool = False
+        self._stick_event_subscribers: dict[
+            Callable[[], None],
+            tuple[Callable[[StickEvent], Awaitable[None]], tuple[StickEvent, ...]],
+        ] = {}
+        self._unsubscribe_stick_events: Callable[[], None] | None = None
+
+    @property
+    def queue_depth(self) -> int:
```

Ruff (0.11.9)

Missing docstring in public method

(D102)